go/bundle/cmd/sign-bundle/integrityblock.go

```diff
@@ -24,7 +24,7 @@ func writeOutput(bundleFile io.ReadSeeker, integrityBlockBytes []byte, originalI
 	return nil
 }
 
-func SignIntegrityBlock(privKey crypto.PrivateKey) error {
+func SignWithIntegrityBlock(privKey crypto.PrivateKey) error {
```

Comment:
Maybe it's good idea to add a function comment, since this is a public (upper-cased) function.

Reply:
Done!